nuclear.js

```diff
@@ -1,16 +1,20 @@
-import { BehaviorContext } from "./Core/Behavior";
+import {BehaviorContext, BehaviorType} from "./Core/Behavior";
 import BehaviorBuilder from "./Core/BehaviorBuilder";
 import objMgr, { me } from "./Core/ObjectManager";
 import { flagsComponents } from "./Core/Util";
+import Heal from "./Targeting/Heal";
 
 export let availableBehaviors = [];  // Declare global variable for behaviors
-
+export let rootBehavior = null; // Declare global variable for rootBehavior
+export let currentBehaviorTypes = [];
+export let HEAL = [];
```

Comment:
No, instead in `Heal.js` export `defaultHealTargeting` that behaviors can import and this file can update